### ISSUE TITLE
Angular 18 support and chart.js interop in v6.0.1

### ISSUE THREAD

**Issue by vzakharov-rxnt** [valor-software/ng2-charts#1981]
**Reproduction of the problem**

I am unable to build a project using ng-charts when Angular is bumped to 18 from 17.
Here is the error:
![image](https://github.com/valor-software/ng2-charts/assets/85562318/88e21a47-8cb2-4a26-9a19-d5372f599cd3)

As you can see, the error is inside `ng-charts.provider.d.ts` and is not related to my code in any way.

There is no such thing as dist/types:
![image](https://github.com/valor-software/ng2-charts/assets/85562318/fb8b4c83-61fa-41ef-97e0-00e2e20b100d)

There is a file called utils.d.ts under here: chart.js\types\
![image](https://github.com/valor-software/ng2-charts/assets/85562318/dc9ad597-1ed3-43a3-87a2-5de4e10b20d3)

Have you tried using Angular 18 with ng-charts? It seems that a bump from 17->18 was the most breaking, we had other issues as well.

**Comment by vzakharov-rxnt** [comment]
This version seems to have no such issues (but ofc we have other build issues from other vendors).
```
"ng2-charts": "5.0.4",
```

**Comment by angelika** [comment]
Any plans to update deps to match Angular 18?

**Comment by focTZi** [comment]
> Any plans to update deps to match Angular 18?

Can you clearify where you are need matching deps? I'm using ng2-charts with angular 18 because the `peerDependencies` are using `>=17.0.0` and not `^` or `~`.

**Comment by angelika** [comment]
You are right, my bad. I must have been using an old version when updating to a newer angular version.
